custom_components/maxcul/__init__.py

```diff
@@ -80,7 +86,7 @@ def _service_enable_pairing(service: ServiceCall):
         service_device_path = service.data[SERVICE_CONF_DEVICE_PATH]
         duration = service.data[SERVICE_CONF_DURATION]
 
-        if service_device_path != device_path:
+        if service_device_path.replace('telnet://','') != device_path.replace("telnet://",""):
```

Comment:
Can you please use single quotes for both parts? I know it is not important, but it looks a bit weird. :wink:

Reply:
Done.